draft-ietf-core-dns-over-coap.md

```diff
@@ -732,6 +733,39 @@ Description: DNS over CoAP resource.
 
 Reference: \[RFC-XXXX\], {{sec:doc-server-selection}}
 
+Operational Considerations
+==========================
+
+Many other DNS transports may co-exist on the DoC server, such as DNS over UDP {{-dns}}, DNS over (D)TLS {{-dot}} {{-dodtls}}, DNS over HTTPS {{-doh}}, or DNS over QUIC {{-doq}}.
+The choice of preference which one to advertise or which one to configure is very use case dependent.
+Protected transports employing transport or object security should be preferred.
+In constrained scenarios with small link layer PDUs, DNS over CoAP should take preference over DNS over DTLS.
+
+At the time of writing, CoAP does not support redirections.
+Nevertheless, changing DNS servers may lead to different results, e.g., with split horizon DNS
+configurations {{?RFC6950}}.
+Any general considerations on a potential CoAP redirect or changing DNS servers apply.
+
+CoAP over unreliable transport layers such as UDP might support different message
+types for Confirmable or Non-Confirmable messages {{-coap}}.
+This document does not enforce any of these message types, as they are very dependent on the
+circumstances of the network and transport chosen for CoAP.
+As DNS over UDP also works over unreliable transports, DNS clients are usually aware, that they might
+never receive a response to a query even after many retries.
+Non-Confirmable messages just reduce the number of such retries to one.
+
+A careless administration might lead to CoAP proxies forming infinite loops.
+The CoAP Hop-Limit option {{-coap-hop-limit}} helps to mitigate such loops.
+
+{{sec:resp-codes}} specifies that DNS operational errors should be reported back to a DoC client
+using the appropriate DNS RCODE.
+If a DoC client did not receive any successful DNS message from a DoC server for a while, it might
+indicate that the DoC server lost connectivity to the upstream DNS infrastructure.
+This should be handled like any recursive resolver that lost connectivity to the upstream DNS
```

Comment:
what does "this" refer to? it is also unclear who should handle this. you probably mean

The DoC client should handle this error case like a recursive resolver that lost connectivity to the upstream DNS infrastructure.